packages/beacon-node/src/network/gossip/encoding.ts

```diff
@@ -53,7 +53,7 @@ export function msgIdFn(gossipTopicCache: GossipTopicCache, msg: GossipsubMessag
     }
   }
 
-  return digest(Buffer.concat(vec)).slice(0, 20);
+  return Buffer.from(digest(Buffer.concat(vec))).slice(0, 20);
```

Comment:
`Buffer.slice()` is deprecated in Node.js 18, should we migrate similar pattern to `Uint8Array.subarray()`?

Reply:
Yes